src/app/Instance/InstancesListPage/InstancesListPage.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
+import React, { CSSProperties, useState } from "react";
```

Comment:
I think we should split the page components from the UI components we plan to move.
Like `InstancesListPage` and `InstancesList`. The first one will stay here and take care of logic and data fetch. The second one will be the UI component moved over (table + paginations).
Does it make sense?

Reply:
I think that this topic deserves a separate meeting. Please ping me when you're available.